ras/servicelog.py

```diff
@@ -118,14 +118,14 @@ def test_servicelog_notify(self):
         self.log.info("=====6 - Checking if servicelog_notify --list, "
                       "lists the command just added  =======")
         for id in ids:
-            if id:
+            if id and not id.isspace():
```

Comment:
function calling in like i suggested in 115 just strip will work so that way eliminate two places to have check